### DISCUSSION TITLE
Momentary On Relay With Tasmota?

### DISCUSSION THREAD

**Discussion by quefol** [arendst/Tasmota#14949] [Q&A] [answered]
I am new to Tasmota. I have it installed on a wemos d1 mini with a relay connected to pin D1. In the WebUI I want to toggle to on and have it briefly pulse on and off but keep it in the on state untill it is trigered again to pulse once more and then go to the off state. Is this even possible? I want to remote power on my 3d printer with a program called octoprint that has a Tasmota pluging to do this. The only problem is Tasmota dose not update the status to the program so it may never knows what actual state it is in. I have tried using PulseTime in Tasmota and it turns on fine but when I try to turn it off it dose not. Possible because Tasmota is already in the off state after it finished pulsing. Just wondering if anyone had ideas on how I can do this. 

Thank you

**Comment by sfromis** [comment, marked as answer]
The general answer for more customized control, when it is more than just a pulse, is to use [rules](https://tasmota.github.io/docs/Rules/).
